src/Schema/IndexDefinition.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Colopl\Spanner\Schema;
+
+use Illuminate\Support\Fluent;
+
+/**
+ * @method $this interleave(string $table)
+ * @method $this nullFiltered()
+ * @method $this storing(string[] $columns)
```

Comment:
Unrelated to this PR, but It might be nice to explain the nullFiltered and storing index in README.md

Reply:
docs have been added.

Reply:
Perfect, thank you.